common/app/common/NewNavigation.scala

```diff
@@ -0,0 +1,83 @@
+package common
+
+sealed class TopLevelSection(val name: String)
+
+case object News extends TopLevelSection("news")
+case object Opinion extends TopLevelSection("opinion")
+case object Sports extends TopLevelSection("sports")
+case object Arts extends TopLevelSection("arts")
+case object Life extends TopLevelSection("life")
```

Comment:
nitpick: I usually like to wrap data type objects/classes like above in an object in order to make the structure and relation between them clearer. 
It also prevents congesting the namespace (especially in common package). Here objects like `News` or `Opinion` for instance are introduced and accessible from everywhere in common package. Wrapping them together would add context.
```
object TopLevelSection {
    case object News extends TopLevelSection("news")
    case object Opinion extends TopLevelSection("opinion")
    case object Sports extends TopLevelSection("sports")
    case object Arts extends TopLevelSection("arts")
    case object Life extends TopLevelSection("life")
}
```